calyx-py/calyx/queues.py

```diff
@@ -19,19 +22,19 @@ def __init__(self, max_len: int):
         self.data = []
         self.max_len = max_len
 
-    def push(self, val: int) -> None:
+    def push(self, val: int, time=0, rank=None) -> None:
```

Comment:
Just stylistically I don't love these unused variables. Can we massage them away? And if we can't, could we preface them with `_` so that they are flagged as unused? So, `_time`.

Reply:
An alternative (which I should have considered earlier) is to use `*args` for the work-conserving algorithms, since this can give us flexibility even later on if we need to deal with more parameters, without ever having to worry about earlier queue structures.

They'll still be unused, but we don't have to worry about them, and we'll only have a max of one 'unused argument'  per function.